content/other/timeline.md

```diff
@@ -0,0 +1,56 @@
++++
++++
+
+# Upcoming activities
+
+> Note:
+>
+> This is a rough plan of what our activities this in Fall 2021, to be updated
+> for the Spring in December. Please let us know if you can suggest things to
+> add here. The best way to do this is by creating an issue at
+> https://github.com/nordic-rse/NordicRSE-organization/issues.
+> Pull request to https://github.com/nordic-rse/nordic-rse.github.io are also
+> welcome.
+>
+
+
+### July
+
+- International RSE survey starts.
+  - Nordic RSE will advertise and increase participation in the Nordics.
+- Add this timeline on the Nordic-RSE website.
+- Proposal to change the Finnish ranking of the [Journal of Open Source Software](https://joss.theoj.org/) from [JUFO ranking](https://www.tsv.fi/julkaisufoorumi/haku.php?lang=en) 0 to 1.
+
+### August
+
+- The first event in the Nordic RSE seminar series.
+- Add some useful links to the website, including:
+  - Hands-on Scientific Computing.
+  - The Aalto RSE career structure.
+  - List of journals publishing open source software.
+
+### September
+
+- Create a how to page for starting an RSE group.
+- Start surveying and contacting RSE-like groups in Nordic Universities.
+
+- 2.9 at 14:00 - 16:00 CEST, 15:00 - 17:00 EEST: First Nordic RSE annual meeting ([public calendar](https://calendar.google.com/calendar/embed?src=8li6hjcjm95g76pgte1p5pi05c%40group.calendar.google.com&ctz=Europe%2FStockholm), [public iCal feed](https://calendar.google.com/calendar/ical/8li6hjcjm95g76pgte1p5pi05c%40group.calendar.google.com/public/basic.ics))
+  - Approve statutes
+  - Select board
+  - Authorize the board to register the association
+
+- First board meeting
+  - submit registration form for the association
+
+### October
```

Comment:
should the international RSE day be mentioned here (14.10.2021)?

Reply:
Yes. I'm on vacation but will get back to it. (but feel welcome to add it if you have time)

Reply:
I guess that would require merging and creating a new PR.